docs/submission/submitting-molecular-data.md

```diff
@@ -145,20 +230,23 @@ Use the returned `analysis_id` to generate a manifest for file upload using the
 Wrote manifest file 'tumor_manifest.txt' for analysisId 'a4142a01-1274-45b4-942a-01127465b422'
 ```
 
-### 3. Upload sequencing files
+### 4. Upload sequencing files
 
-Using the score-client `upload` command, upload all files associated with the payload.
+Using the score-client `upload` command, upload all files associated with the payload. This requires the manifest file generated in step 3.
 
 ```
 .bin/score-client  upload --manifest manifest.txt
 ```
 
 If the file successfully uploads, then you will receive an `Upload completed` message.
 
-### 4. Publish the payload
+### 5. Publish the payload
 
 ```
+// need publish command
 AnalysisId a4142a01-1274-45b4-942a-01127465b422 successfully published
 ```
 
+Once your `sequencing_experiment` analysis has been successfully submitted, it will be queued for data processing. You can follow the progress of submitted data on your [Program Dashboard](../submission/submitted-data).

```

Comment:
do we want to say molecular anywhere here?
`You can follow the progress of molecular data processing on your`...